tests/_support/Traits/With_Uopz.php

```diff
@@ -0,0 +1,205 @@
+<?php
+
+namespace StellarWP\Telemetry\Tests\Support\Traits;
+
+use PHPUnit\Framework\Assert;
+
+trait With_Uopz {
+	/*
+	 * The following properties are static to cover data providers where 2 diff. instances of the test case are used:
+	 * one to build the data sets, the other to run the tests.
+	 */
+	private static $uopz_set_returns = []; private static $uopz_redefines = [];
+	private static $uopz_set_properties = [];
+	private static $uopz_add_class_fns = [];
+	private static $uopz_del_functions = [];
+
+	/**
+	 * @after
+	 */
+	public function unset_uopz_returns() {
+		if ( function_exists( 'uopz_set_return' ) ) {
+			foreach ( self::$uopz_set_returns as $f ) {
+				if ( is_array( $f ) ) {
+					list( $class, $method ) = $f;
+					uopz_unset_return( $class, $method );
+				} else {
+					uopz_unset_return( $f );
+				}
+			}
+		}
+
+		self::$uopz_set_returns = [];
+	}
+
+	/**
+	 * @after
+	 */
+	public function unset_uopz_redefines() {
+		if ( function_exists( 'uopz_redefine' ) ) {
+			foreach ( self::$uopz_redefines as $restore_callback ) {
+				$restore_callback();
+			}
+		}
+
+		self::$uopz_redefines = [];
+	}
+
+	/**
+	 * @after
+	 */
+	public function unset_uopz_properties() {
+		if ( function_exists( 'uopz_set_property' ) ) {
+			foreach ( self::$uopz_set_properties as $definition ) {
+				list( $object, $field, $original_value ) = $definition;
+				// Overwrite value with what we stored as the original value.
+				uopz_set_property( $object, $field, $original_value );
+			}
+		}
+		self::$uopz_set_properties = [];
+	}
+
+	/**
+	 * @after
+	 */
+	public function unset_uopz_functions() {
+		if ( function_exists( 'uopz_del_function' ) ) {
+			foreach ( self::$uopz_del_functions as $function ) {
+				uopz_del_function( $function );
+			}
+		}
+
+		self::$uopz_del_functions = [];
+	}
+
+	/**
+	 * Wrapper for uopz_set_return
+	 *
+	 * @since 4.15.1
```

Comment:
Was this copied from somewhere else? 
```suggestion
	 * @since TBD
```

Reply:
Yes, since it's test code, I'm removing all the `@since` annotations.